Apps/WebClient/src/ClientApp/src/store/modules/vaccinationStatus/actions.ts

```diff
@@ -94,8 +97,28 @@ export const actions: VaccinationStatusActions = {
                     params.dateOfVaccine
                 )
                 .then((result) => {
+                    const payload = result.resourcePayload;
                     if (result.resultStatus === ResultType.Success) {
-                        const payload = result.resourcePayload;
+                        context.commit("setPublicVaccineRecord", payload);
+                        resolve(payload);
+                    } else if (
+                        result.resultError?.actionCode === ActionType.Refresh &&
+                        !payload.loaded &&
+                        payload.retryin > 0
+                    ) {
+                        logger.info("Public VaccinationStatus not loaded");
```

Comment:
The logger message should reference the Vaccine Proof rather than the VaccinationStatus.